runtime/queries/svelte/highlights.scm

```diff
@@ -0,0 +1,68 @@
+; Special identifiers
+;--------------------
+
+; TODO:
+((element (start_tag (tag_name) @_tag) (text) @text.title)
```

Comment:
Note: these text tags should likely be https://www.sublimetext.com/docs/scope_naming.html#markup but can be changed in a follow-up since there's a few other grammars to fix too.

Reply:
I've changed to the related markup tags